addons/test_mail/tests/test_performance.py

```diff
@@ -166,14 +166,16 @@ def test_adv_activity_full(self):
             'default_res_model': 'mail.test.activity',
         })
 
-        with self.assertQueryCount(margin=1, admin=10, emp=15):  # test_mail only: 9 - 13
+        with self.assertQueryCount(admin=10, emp=15):  # test_mail only: 9 - 13
             activity = MailActivity.create({
                 'summary': 'Test Activity',
                 'res_id': record.id,
                 'activity_type_id': self.env.ref('mail.mail_activity_data_todo').id,
             })
-
-        with self.assertQueryCount(margin=1, admin=27, emp=47):  # test_mail only: 27 - 47
+            category = activity.activity_type_id.category
+            #read activity_type to normalize cache between enterprise and community
+            #voip module read activity_type during create leading to one less query in enterprise on action_feedback
+        with self.assertQueryCount(admin=26, emp=46, query_test="action_feedback"):  # test_mail only: 27 - 47
```

Comment:
Just for the sake of readability, please leave the blank line between `with` statements.
Also please put the comment before the line it documents.